lib/index.js

```diff
@@ -289,31 +274,42 @@ class CorePlugin {
   createComputedField (request) {
     if (request.input.body 
       && request.input.body.name
-      && request.input.body.index
-      && request.input.body.collection
+      && request.input.resource.index
+      && request.input.resource.collection
       && request.input.body.value
     ) {
-      return this.addComputedField(request.input.body)
+      return this.addComputedField(request.input.resource.index, request.input.resource.collection, request.input.body)
         .then(res => {
           return this.loadComputedFields().then(() => res) 
         })
     }
-    return Promise.reject('Body must contain the following fields: \'name\', \'index\', \'collection\' and \'value\'')
+    return Promise.reject('Body must contain the following fields: \'name\' and \'value\'')
   }
 
   deleteComputedField (request) {
-    let _id = request.input.args._id || request.input.resource._id
-    if (_id) {
-      return this.removeComputedField(_id)
+    let index = request.input.resource.index
```

Comment:
should be `const`

Alternatively, you can use destructuring syntax: 
`const {index, collection} = request.input.resource;`